.github/workflows/dockerize.yaml

```
@@ -0,0 +1,30 @@
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6
+        with:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 25: did not find expected key

(syntax-check)